inception-active-learning/src/main/java/de/tudarmstadt/ukp/inception/active/learning/sidebar/ActiveLearningSidebar.java

```diff
@@ -589,15 +602,20 @@ public void afterAnnotationUpdateEvent(AjaxAfterAnnotationUpdateEvent aEvent)
         AnnotatorState annotatorState = getModelObject();
         AnnotatorState eventState = aEvent.getAnnotatorState();
 
-        //check active learning is active and same user and same document and same layer
-        if (sessionActive && eventState.getUser().equals(annotatorState.getUser()) && eventState
-            .getDocument().equals(annotatorState.getDocument()) && annotatorState
-            .getSelectedAnnotationLayer().equals(selectedLayer.getObject())) {
+        // Check active learning is active and same user and same document and same layer
+        if (
+                sessionActive && 
+                currentRecommendation != null &&
+                eventState.getUser().equals(annotatorState.getUser()) && 
+                eventState.getDocument().equals(annotatorState.getDocument()) && 
+                annotatorState.getSelectedAnnotationLayer().equals(selectedLayer.getObject())
+        ) {
             //check same document and same token
             if (annotatorState.getSelection().getBegin() == currentRecommendation.getOffset()
                 .getBeginCharacter()
                 && annotatorState.getSelection().getEnd() == currentRecommendation.getOffset()
-                .getEndCharacter() && aEvent.getValue() != null) {
+                .getEndCharacter()
```

Comment:
I think  `aEvent.getValue() != null` should be kept. 
If `aEvent.getValue()` of `AjaxAfterAnnotationUpdateEvent` is null, it means that a user has selected the token in the mainEditor, but hasn't made any annotation. So at this time, AL sidebar doesn't need to update the suggestion.